primer.html

```diff
@@ -502,12 +505,32 @@ <h2>Updating ShapeTree Instances <a class="cross" href="spec#post-managed"></a><
     tree:shapeTreeInstanceRoot &lt;&gt;.
           </code></pre>
     </section>
-    <!-- ul>
-        <li><b>PhotoAlbum</b> - collects photos or references to photos into sharable albums (ShapeTree <a href="https://github.com/janeirodigital/shapetrees/blob/master/solidApps/staticRoot/album/PhotoAlbumShapeTree.ttl">Turtle</a> / <a href="https://github.com/janeirodigital/shapetrees/blob/master/solidApps/staticRoot/album/PhotoAlbumShapeTree.jsonld">JSON-LD</a>, <a href="https://github.com/janeirodigital/shapetrees/blob/master/solidApps/staticRoot/album/PhotoAlbum.shex">schema</a>).</li>
-        <li><b>MultiCal</b> - migrates calendar applications between two common formats, "Common" (ShapeTree <a href="https://github.com/janeirodigital/shapetrees/blob/master/solidApps/staticRoot/cal/CalendarShapeTree.ttl">Turtle</a> / <a href="https://github.com/janeirodigital/shapetrees/blob/master/solidApps/staticRoot/cal/CalendarShapeTree.jsonld">JSON-LD</a>, <a href="https://github.com/janeirodigital/shapetrees/blob/master/solidApps/staticRoot/cal/Calendar.shex">schema</a>), and "Google" (ShapeTree <a href="https://github.com/janeirodigital/shapetrees/blob/master/solidApps/staticRoot/cal/GoogleShapeTree.ttl">Turtle</a> / <a href="https://github.com/janeirodigital/shapetrees/blob/master/solidApps/staticRoot/cal/GoogleShapeTree.jsonld">JSON-LD</a>, <a href="https://github.com/janeirodigital/shapetrees/blob/master/solidApps/staticRoot/cal/GoogleCalendar.shex">schema</a>).</li>
-        <li><b>GitHub</b> - a storage repository associating users or groups with repositories, issues and events (ShapeTree <a href="https://github.com/janeirodigital/shapetrees/blob/master/solidApps/staticRoot/gh/ghShapeTree.ttl">Turtle</a> / <a href="https://github.com/janeirodigital/shapetrees/blob/master/solidApps/staticRoot/gh/ghShapeTree.jsonld">JSON-LD</a>, <a href="https://github.com/janeirodigital/shapetrees/blob/master/solidApps/staticRoot/gh/ghSchema.shex">schema</a>).</li>
-      </ul -->
 
+    <section id="describing">
+      <h2>Describing Shape Trees <a class="cross" href="spec#describing-shapetrees"></a></h2>
+      <p>
+        To aid in the human-readability of Shape Trees, a SKOS graph can be used to provide a textual representation of the structure.
+      </p>
+      <p>
+        Making use of the <code>tree:step</code> predicate, labels can be applied to specific steps in a Shape Tree using any language.
+        A SKOS index document allows for discovery of multiple definitions of a Shape Tree supporting multiple language preferences and cognitive abilities.
```

Comment:
How does an automated agent determine the hierarchy for english, spanish, or the english hierarchy for people with diminished cognitive ability? If it's expected that those rules should come from the ecosystem in which they are applied, I think we should at least state that here.

Reply:
Added that context.